### ISSUE TITLE
when -r is specified, the read count output includes forward and reverse reads

### ISSUE THREAD

**Issue by wwood** [geronimp/graftM#6]
The user would expect this number to be the number of pairs, not the number of individual reads, at least how the user reads.
```
[10:46:37]: Found 57127 read(s) in cck10_for.
```
I think a best solution would be to report the number of forward reads separately to the number of reverse reads found

**Comment by wwood** [comment]
Joel will check if already done?

**Comment by geronimp** [comment]
This was fixed as of 0.7.0:
```
                             GraftM 0.7.0

                                GRAFT

                       Joel Boyd, Ben Woodcroft

                                                         __/__
                                                  ______|
          _- - _                         ________|      |_____/
           - -            -             |        |____/_
           - _     >>>>  -   >>>>   ____|
          - _-  -         -             |      ______
             - _                        |_____|
           -                                  |______
            
09/28/2015 05:50:58 PM INFO: Working on 20120600_E1D.1
09/28/2015 05:50:58 PM INFO: Working on forward reads
09/28/2015 05:54:53 PM INFO: 4583 read(s) detected
09/28/2015 05:54:53 PM INFO: Aligning reads to reference package database
09/28/2015 05:55:13 PM INFO: Working on reverse reads
09/28/2015 05:58:25 PM INFO: 4485 read(s) detected
```
